.github/workflows/ci.yml

```diff
@@ -79,6 +128,7 @@ jobs:
 
   event_schema:
     name: Event Schema
+    if: "!startsWith(github.ref , 'refs/heads/release')"
```

Comment:
I assume we want to avoid rebuilding for no purpose when releasing but I would just keep it if it doesn't hurt (unless this builds on all platforms for whatever reason?)

Reply:
This only builds on ubuntu-latest, but I'm trying to cut down on all non-essential jobs to make releases more reliable since they are unattended. Was even thinking of skipping lints.